ruby/vampires.rb

```diff
@@ -82,6 +83,7 @@
 	if name == ("Drake Cula" || "Tu Fang")

```

Comment:
In this case if the name is Fang or Drake then we know this person is a vampire so logically this would be the first thing you should consider checking. Is there any reason to check anymore of the code if we know they are a vampire already?